angular/core/components/group_page/group_page_controller.coffee

```diff
@@ -61,16 +61,12 @@ angular.module('loomioApp').controller 'GroupPageController', ($rootScope, $loca
       ModalService.open SubscriptionSuccessModal
 
   @showWelcomeModel = ->
-    @group.isParent() and
-    AbilityService.isCreatorOf(@group) and
-    @group.noInvitationsSent() and
-    !@group.trialIsOverdue() and
-    !@subscriptionSuccess and
```

Comment:
I think you may still need this subscription success one, since the subscription modal wants precedence over the welcome modal